cupy_backends/cuda/libs/cusparselt.pyx

```diff
@@ -0,0 +1,365 @@
+"""Thin wrapper for cuSPARSELt"""
+
+cimport cython  # NOQA
+
+from cpython.mem cimport PyMem_Malloc, PyMem_Free
+from libc.stdint cimport uint8_t, int32_t, uint32_t, int64_t, intptr_t
+
+from cupy_backends.cuda cimport stream as stream_module
+from cupy_backends.cuda.api cimport driver
+
+from cupy_backends.cuda.libs import cusparse
+
+cdef extern from '../../cupy_cusparselt.h' nogil:
+    ctypedef int cusparseStatus_t 'cusparseStatus_t'
+    ctypedef int cusparseOrder_t 'cusparseOrder_t'
+    ctypedef int cudaDataType 'cudaDataType'
+    ctypedef int cusparseComputeType 'cusparseComputeType'
+
+    # Opaque Data Structures
+    ctypedef struct cusparseLtHandle_t 'cusparseLtHandle_t':
+        uint8_t data[1024]
+    ctypedef struct cusparseLtMatDescriptor_t 'cusparseLtMatDescriptor_t':
+        uint8_t data[1024]
+    ctypedef struct cusparseLtMatmulDescriptor_t 'cusparseLtMatmulDescriptor_t':  # NOQA
+        uint8_t data[1024]
+    ctypedef struct cusparseLtMatmulAlgSelection_t 'cusparseLtMatmulAlgSelection_t':  # NOQA
+        uint8_t data[1024]
+    ctypedef struct cusparseLtMatmulPlan_t 'cusparseLtMatmulPlan_t':
+        uint8_t data[1024]
```

Comment:
Nitpick: one trick done elsewhere in the codebase to avoid lengthy lines (for which you had to add NOQA) is to strip away the library prefix. Also, do we need to know the struct layout in Cython? Looks to me they're implementation detail that is not used. If so, you can simplify it greatly: 
```suggestion
    # Opaque Data Structures
    ctypedef struct Handle 'cusparseLtHandle_t':
        pass
    ctypedef struct MatDescriptor 'cusparseLtMatDescriptor_t':
        pass
    ctypedef struct MatmulDescriptor 'cusparseLtMatmulDescriptor_t':
        pass
    ctypedef struct MatmulAlgSelection 'cusparseLtMatmulAlgSelection_t':
        pass
    ctypedef struct MatmulPlan 'cusparseLtMatmulPlan_t':
        pass
```
or even better? (No `pass` needed)
```
ctypedef void* Handle 'cusparseLtHandle_t'
...
```